src/componentsEventPage/CategoryFilter.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+
+export default function CategoryFilter ({ onSelectFilter }) {
+  
+  return (
+     <FilterContainer>
+      <CategoryButton name="metal"  onClick={() => onSelectFilter('metal')}>metal</CategoryButton>
+      <CategoryButton name="medieval" onClick={() => onSelectFilter('medieval')}>medieval</CategoryButton>
+      <CategoryButton name="sand"  onClick={() => onSelectFilter('sand')}>sand</CategoryButton>  
+      <CategoryButton name="others"  onClick={() => onSelectFilter('others')}>others</CategoryButton>
```

Comment:
Maybe add a default value, that lines up with your state on `eventPage`?

Reply:
I will check this, I have a button "all", which shows all lists, if nothing is filtered, so I thought this is also my default value.  - the default button is shown in the header directly, not as a CategoryButton, maybe I need to change this